src/components/v5/common/ActionSidebar/partials/Motions/steps/FinalizeStep/FinalizeStep.tsx

```diff
@@ -84,7 +85,8 @@ const FinalizeStep: FC<FinalizeStepProps> = ({
   };
   if (
     actionData.motionData.isFinalized ||
-    actionData.motionData.motionStateHistory.hasFailedNotFinalizable
+    actionData.motionData.motionStateHistory.hasFailedNotFinalizable ||
+    actionData.type === ColonyActionType.CreateDecisionMotion
```

Comment:
maybe we could put this (perhaps some other too) into boolean variables for improved readability